sharpy/aero/utils/utils.py

```diff
@@ -121,3 +121,55 @@ def span_chord(i_node_surf, zeta):
     dir_chord = algebra.unit_vector(dir_chord)
 
     return dir_span, span, dir_chord, chord
+
+
+def find_aerodynamic_solver(settings):
```

Comment:
This is amendable for a general solution:
solver_interface.dictionary_of_solvers  -> to give you the solvers an loop over them 
if solver_i.solver_classification == 'Aero': 
    ... (same structure as in the code)
elif solver_i.solver_classification == 'Coupled: 
    ...

Worthy? Only if more solvers are to be added in the equilibrium solution. But it shows off the nice structure of the solvers class!!